VRUI.java

```diff
@@ -0,0 +1,224 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+public class VRUI {
+	private static Scanner scanner = new Scanner(System.in) ;
+
+	private List<Customer> customers = new ArrayList<Customer>() ;
+	
+	private List<Video> videos = new ArrayList<Video>() ;
+	
+	public static void main(String[] args) {
+		VRUI ui = new VRUI() ;
+		
+		boolean quit = false ;
+		while ( ! quit ) {
+			int command = ui.showCommand() ;
+			switch ( command ) {
+				case 0: quit = true ; break ;
+				case 1: ui.listCustomers() ; break ;
+				case 2: ui.listVideos() ; break ;
+				case 3: ui.register("customer") ; break ;
+				case 4: ui.register("video") ; break ;
+				case 5: ui.rentVideo() ; break ;
+				case 6: ui.returnVideo() ; break ;
+				case 7: ui.getCustomerReport() ; break; 
+				case 8: ui.clearRentals() ; break ;
+				case -1: ui.init() ; break ;
+				default: break ;
+			}
+		}
+		System.out.println("Bye");
+	}
+
+	public void clearRentals() {
+		System.out.println("Enter customer name: ") ;
+		String customerName = scanner.next() ;
+		
+		Customer foundCustomer = null ;
+		for ( Customer customer: customers ) {
+			if ( customer.getName().equals(customerName)) {
+				foundCustomer = customer ;
+				break ;
+			}
+		}
+
+		if ( foundCustomer == null ) {
+			System.out.println("No customer found") ;
+		} else {
+			System.out.println("Name: " + foundCustomer.getName() +
+					"\tRentals: " + foundCustomer.getRentals().size()) ;
+			for ( Rental rental: foundCustomer.getRentals() ) {
+				System.out.print("\tTitle: " + rental.getVideo().getTitle() + " ") ;
+				System.out.print("\tPrice Code: " + rental.getVideo().getPriceCode()) ;
```

Comment:
I think adding another  `+ " \n"` at the end of the code, just like line 54, would make it look nicer :)